ssl/statem/statem_clnt.c

```diff
@@ -1036,7 +1036,7 @@ int tls_construct_client_hello(SSL *s, WPACKET *pkt)
             }
         }
     } else
-        i = 1;
+        i = s->hello_retry_request == 0;
```

Comment:
While we are touching this line anyway, could we enclose it in `{}`. The style guide says that if one branch of an `if` uses `{}` then both should.

Reply:
"{}" added in 5187e42